app/static/css/style.css

```diff
@@ -103,10 +109,17 @@ html, body {
 }
 
 .navbar-brand img {
-  width: 400px;
-  padding: 5px;
+  width: 350px;
+  padding: 0px;
+}
+.sticky-top {
+  position: fixed;
+  top: 0;
+  width: 100%;
+}
+#id {
```

Comment:
This will not target anything in the html unless an `id` is set to `id`, so it should probably be removed as redundant.